### PR TITLE
Rename receive event data property

### PR DESCRIPTION
**Pull request by Or-Geva**
To ensure consistent handling of various message types, the receiver events in the web view have been refactored to follow a standardized JSON structure:

```json
{
"type": "<value>",
"data": "<value>"
}
```
 This allows for easier handling of different event notifications received by the web view.
 
 Related to:
 * https://github.com/jfrog/jfrog-idea-plugin/pull/370

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,7 +56,7 @@ To view a specific page example, choose one of the following options and enter t
 ```javascript
 window.postMessage({
```
Comment (yahavi) [inline README.md:57]: The three JSONs in the README are not aligned properly. Let's fix the indentation to make them easy to read.

File: src/App.test.tsx [hunk 2]
```diff
@@ -12,17 +12,17 @@ import { ISeverity } from './model/severity'
...
 
 describe('App component', () => {
 	test('renders the Dependency page when the page type is "Dependency"', async () => {
 		render(<App />)
 
 		// Load Dependency.
-		const pageData = {
-			type: WebviewReceiveEventType.ShowPage,
-			pageData: {
+		const mock = {
```
Comment (yahavi) [inline src/App.test.tsx:23]: "mock" is very abstract. Please be more specific when naming variables.
```suggestion
		const mockPage = {
```
(Let's do this change in all other locations)

File: src/components/UI/Modal/pages/LoginModal.tsx [hunk 3]
```diff
@@ -73,21 +73,25 @@ function createBody(data: ILoginPage): JSX.Element {
...
 			pageBody = (
-				<div>
-					<span>We found a JFrog CLI with connection details for </span>
-					<span className={css.textBold}>{data.url}</span>.
+				<div className={css.autoLoginText}>
+					<span> It looks like JFrog CLI is installed with the connection details of</span>
+					<div>
+						<span className={css.textBold}>{data.url}</span>
+					</div>
 					<div />
```
Comment (yahavi) [inline src/components/UI/Modal/pages/LoginModal.tsx:81]: Is the empty div by purpose?

**Review by yahavi — CHANGES_REQUESTED** [review]

File: src/components/UI/Modal/pages/LoginModal.tsx (hunk 3 above)
Comment (Or-Geva) [reply src/components/UI/Modal/pages/LoginModal.tsx:81]: Yes, to separate sentences

**Pull request merged**